nilearn/plotting/surf_plotting.py

```diff
@@ -422,6 +422,262 @@ def plot_surf_stat_map(surf_mesh, stat_map, bg_map=None,
     return display
 
 
+def _check_display_mode(display_mode):
+    """Checks whether the display_mode string passed to plot_img_on_surf
+    is meaningful.
+
+    display_mode: :obj:`str`
+        Any combination of 'lateral', 'medial', 'dorsal', 'ventral',
+        'anterior', 'posterior', united by a '+'. Ex.: 'dorsal+lateral'
```

Comment:
General UX question: Why are we requiring a string rather than a list, here ? I can't think of another example where we format arguments in this way ?

Reply:
Sounds good. What do you think about creating an *Orientation* `Enum` specifying the options along with a function that normalizes string input to return the selection? I think it could be an improvement.